src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -570,14 +570,16 @@ private void prepareMetricStatements() {
                 "INSERT INTO reaper_db.running_repairs(repair_id, node)"
                     + "values (?, ?) IF NOT EXISTS")
             .setSerialConsistencyLevel(ConsistencyLevel.SERIAL)
-            .setIdempotent(true);
+            .setConsistencyLevel(ConsistencyLevel.QUORUM)
+            .setIdempotent(false);
```

Comment:
We've seen that somehow it allowed the driver to perform speculative executions that triggered the following errors:

``` 
Caused by: com.datastax.driver.core.exceptions.InvalidQueryException: SERIAL is not supported as conditional update commit consistency. Use ANY if you mean "make sure it is accepted but I don't care how many replicas commit it for non-SERIAL reads"
        at com.datastax.driver.core.Responses$Error.asException(Responses.java:181)
        at com.datastax.driver.core.DefaultResultSetFuture.onSet(DefaultResultSetFuture.java:215)
        at com.datastax.driver.core.RequestHandler.setFinalResult(RequestHandler.java:235)
        at com.datastax.driver.core.RequestHandler.access$2600(RequestHandler.java:61)
        at com.datastax.driver.core.RequestHandler$SpeculativeExecution.setFinalResult(RequestHandler.java:1011)
        at com.datastax.driver.core.RequestHandler$SpeculativeExecution.onSet(RequestHandler.java:814)
        at com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:1287)
        at com.datastax.driver.core.Connection$Dispatcher.channelRead0(Connection.java:1205)
```

It could have also been the fact that the update CL wasn't explicitly set, but I feel like we don't need those to be idempotent as retries could cause contention and make a bad situation worse.
Missing a lock is mostly fine.